### PR TITLE
refactor: team

### PR DESCRIPTION
**Pull request by rohitp-bsf**
### Description
- refactor: team

### Screenshots
- NA

### Types of changes
- Converted Class-based components to functional-based.
- Refactored the edit JS file into render and settings.

### How has this been tested?
- NA

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/team/render.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+		if ( image.sizes ) {
+			img_url = size[ imgSize ] ? size[ imgSize ].url : image.url;
+		} else {
+			img_url = image.url;
+		}
+	}
+
+	let image_html = '';
+
+	if ( '' != img_url ) {
```
Comment (RajkiranBagal) [inline src/blocks/team/render.js:179]: Use camelCase variables for consistency

File: src/blocks/team/settings.js [hunk 2]
```diff
@@ -0,0 +1,1120 @@
...
+				],
+			},
+		};
+
+		loadDescGoogleFonts = (
+			<WebfontLoader config={ dconfig }></WebfontLoader>
+		);
+	}
+
+	return (
```
Comment (RajkiranBagal) [inline src/blocks/team/settings.js]: Create separate functions for BlockControls & Each panel settings

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/team/settings.js (hunk 2 above)
Comment (rohitp-bsf) [reply src/blocks/team/settings.js]: Done!

File: src/blocks/team/render.js (hunk 1 above)
Comment (rohitp-bsf) [reply src/blocks/team/render.js]: Done!

File: src/blocks/team/save.js [hunk 3]
```diff
@@ -42,93 +46,112 @@ export default function save( props ) {
...
 		socialEnable,
-		stack
-	} = props.attributes
+		stack,
+	} = props.attributes;
 
-	let size = ""
-	let img_url = ""
+	let size = '';
+	let img_url = '';
```
Comment (RajkiranGajananBagal) [inline src/blocks/team/save.js:53]: camelCase variables.

File: src/blocks/team/settings.js [hunk 4]
```diff
@@ -0,0 +1,1119 @@
...
+		}
+		setAttributes( { image: media } );
+
+		if ( media.sizes ) {
+			const new_img = getImageSize( media.sizes );
+			imageSizeOptions = new_img;
+		}
+	};
+
+	const svg_icons = Object.keys( UAGBIcon );
```
Comment (RajkiranGajananBagal) [inline src/blocks/team/settings.js:163]: camelCase variables.

File: src/blocks/team/styling.js [hunk 5]
```diff
@@ -55,159 +53,206 @@ function styling( props ) {
...
 
-	if ( "above" == imgPosition ) {
-		if ( "center" == align ) {
-			selectors[" .uagb-team__image-wrap"]["margin-left"] = "auto"
-			selectors[" .uagb-team__image-wrap"]["margin-right"] = "auto"
-		} else if ( "left" == align ) {
-			selectors[" .uagb-team__image-wrap"]["margin-right"] = "auto"
-		} else if ( "right" == align ) {
-			selectors[" .uagb-team__image-wrap"]["margin-left"] = "auto"
+	let tablet_selectors = {};
```
Comment (RajkiranGajananBagal) [inline src/blocks/team/styling.js:58]: camelCase variables.

**Review by RajkiranGajananBagal — CHANGES_REQUESTED** [review]

File: src/blocks/team/styling.js (hunk 5 above)
Comment (rohitp-bsf) [reply src/blocks/team/styling.js]: done!

File: src/blocks/team/settings.js (hunk 4 above)
Comment (rohitp-bsf) [reply src/blocks/team/settings.js]: done!

File: src/blocks/team/save.js (hunk 3 above)
Comment (rohitp-bsf) [reply src/blocks/team/save.js:53]: done!

File: src/blocks/team/block.js [hunk 6]
```diff
@@ -2,30 +2,26 @@
...
+import attributes from './attributes';
+import './editor.scss';
+import './style.scss';
 
 import { __ } from '@wordpress/i18n';
 
-const {
-	registerBlockType,
-} = wp.blocks
+const { registerBlockType } = wp.blocks;
```
Comment (sandeshjangam) [inline src/blocks/team/block.js:15]: Use package here

File: src/blocks/team/deprecated.js [hunk 7]
```diff
@@ -2,35 +2,48 @@
...
+import classnames from 'classnames';
+import attributes from './attributes';
+import renderSVG from '@Controls/renderIcon';
 
-const {
-	RichText
-} = wp.blockEditor
-
-const { Fragment } = wp.element
+const { RichText } = wp.blockEditor;
```
Comment (sandeshjangam) [inline src/blocks/team/deprecated.js:9]: Use package

File: src/blocks/team/edit.js [hunk 8]
```diff
@@ -2,813 +2,62 @@
...
-import WebfontLoader from "../../components/typography/fontloader"
+const Settings = lazy( () =>
+	import( /* webpackChunkName: "chunks/team/settings" */ './settings' )
+);
+const Render = lazy( () =>
+	import( /* webpackChunkName: "chunks/team/render" */ './render' )
+);
 
-const { withSelect } = wp.data
+const { withSelect } = wp.data;
```
Comment (sandeshjangam) [inline src/blocks/team/edit.js:17]: Use package

File: src/blocks/team/render.js [hunk 9]
```diff
@@ -0,0 +1,230 @@
+import classnames from 'classnames';
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import renderSVG from '@Controls/renderIcon';
+const { createBlock } = wp.blocks;
+const { RichText } = wp.blockEditor;
```
Comment (sandeshjangam) [inline src/blocks/team/render.js:6]: Use package

File: src/blocks/team/save.js [hunk 10]
```diff
@@ -2,25 +2,29 @@
...
-import renderSVG from "@Controls/renderIcon"
+import classnames from 'classnames';
+import renderSVG from '@Controls/renderIcon';
 
-const {
-	RichText
-} = wp.blockEditor
-
-const { Fragment } = wp.element
+const { RichText } = wp.blockEditor;
```
Comment (sandeshjangam) [inline src/blocks/team/save.js:8]: Use package

File: src/blocks/team/settings.js [hunk 11]
```diff
@@ -0,0 +1,1119 @@
...
+	{ value: 'full', label: __( 'Large', 'ultimate-addons-for-gutenberg' ) },
+];
+
+const {
+	AlignmentToolbar,
+	BlockControls,
+	InspectorControls,
+	PanelColorSettings,
+	MediaUpload,
+} = wp.blockEditor;
```
Comment (sandeshjangam) [inline src/blocks/team/settings.js:41]: Use package

File: src/blocks/team/settings.js [hunk 12]
```diff
@@ -0,0 +1,1119 @@
...
+
+const {
+	PanelBody,
+	SelectControl,
+	RangeControl,
+	Button,
+	TextControl,
+	BaseControl,
+	ToggleControl,
+} = wp.components;
```
Comment (sandeshjangam) [inline src/blocks/team/settings.js:51]: Use package

**Review by sandeshjangam — CHANGES_REQUESTED** [review]
Need changes

File: src/blocks/team/settings.js (hunk 12 above)
Comment (rohitp-bsf) [reply src/blocks/team/settings.js]: Done!

File: src/blocks/team/settings.js (hunk 11 above)
Comment (rohitp-bsf) [reply src/blocks/team/settings.js]: Done!

File: src/blocks/team/save.js (hunk 10 above)
Comment (rohitp-bsf) [reply src/blocks/team/save.js]: Done!

File: src/blocks/team/render.js (hunk 9 above)
Comment (rohitp-bsf) [reply src/blocks/team/render.js]: Done!

File: src/blocks/team/edit.js (hunk 8 above)
Comment (rohitp-bsf) [reply src/blocks/team/edit.js]: Done!

File: src/blocks/team/deprecated.js (hunk 7 above)
Comment (rohitp-bsf) [reply src/blocks/team/deprecated.js]: Done!

File: src/blocks/team/block.js (hunk 6 above)
Comment (rohitp-bsf) [reply src/blocks/team/block.js]: Done!

**Review by sandeshjangam — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**